app/views/shared/_date_time_form_part.html.erb

```diff
@@ -4,6 +4,11 @@
     <%- if local_assigns.fetch(:show_label, false) %>
       <label for>
```

Comment:
I know this wasn't change, but I just noticed that we have `label` inside `label`.